pytest_logger/plugin.py

```diff
@@ -32,7 +43,19 @@ def logsdir(self):
         ldir = self._logsdir
         if ldir:
             return ldir
-        ldir = self._logsdir = _make_logsdir(self.config._tmpdirhandler, self.logdirlinks)
+        logger_logdir = self.config.getoption('logger_logdir')
+        if logger_logdir is None or logger_logdir == '':
```

Comment:
If logger_logdir is checked for emptiness and noneness, "if logger_logdir:" would probably be enough.